content/200-concepts/100-components/02-prisma-client/051-working-with-fields/100-working-with-json-fields.mdx

```diff
@@ -874,6 +874,14 @@ To differentiate between these possibilities, we've introduced three _null enums
 - `DbNull`: Selects the `NULL` value in the database.
 - `AnyNull`: Selects both `null` JSON values and `NULL` database values.
 
+<Admonition type="info">
+
+  * When filtering using any of the _null enums_ you can not use a shorthand and leave the `equals` operator off.
```

Comment:
I am not sure if this is exactly correct. @dpetrick?

Reply:
Can be formulated more clearly, but the statement is correct: There are no shorthands for json filtering. The reason is simple: We can't construct a TypeScript API that allows arbitrary JSON and have shorthands at the same time, because you can't differentiate a shorthand from a JSON object if you don't have a fixed depth of the query. E.g. there's no way for us to differentiate if the user wants to use a shorthand or if the user wants to store an object with `{"equals": ...}`.